JosephNjuguna/QuickCredit
# Ch setup project 166114341

### What does this PR do?
- [ ] adds project configuration.
- [ ] initializes node into the project.
- [ ]  adds dotenv file.
- [ ] add configuration for : 
1. babel
2. eslint
- [ ] adds app.js file to start the api and server.js file to start the api server
### Description of the task to be completed?
- [ ] add required project configuration.
### How should this be manually tested?
- [ ] git clone this repo to your machine using ``.
- [ ] open terminal and using `cd` navigate to the folder where you cloned the repo and run 
`npm start`
**the api should start** and log to terminal: **API listening on port 3000`** "
### Any background context you want to provide?
### Relevant pivotal tracker stories ?
- [ ] https://www.pivotaltracker.com/story/show/166114341
